torchdynamo/resume_execution.py

```diff
@@ -29,9 +30,14 @@ class ReenterWith:
 
     def __call__(self, code_options, cleanup):
         with_cleanup_start = create_instruction("WITH_CLEANUP_START")
+        if sys.version_info < (3, 8):
+            begin_finally = create_instruction("LOAD_CONST", 0)
```

Comment:
This is close, but might not be exactly right.  

[LOAD_CONST](https://docs.python.org/3/library/dis.html#opcode-LOAD_CONST) pushes `code_options["co_consts"][inst.arg]` onto the stack.

I think you are assuming `code_options["co_consts"][0] is None`, which might not be true.

You may need to update co_consts similar to this: https://github.com/facebookresearch/torchdynamo/blob/16f6836e2f9793b99f40ee0f047ebc3a46e9e162/torchdynamo/codegen.py#L214
Ideally you could just call that code.

Reply:
Thanks for catching this! I find it's not easy to reuse the ```create_load_const``` as we can't get the ```PyGen``` object in this context, so I write a similar function follow that one. Lmk if it makes sense.